otx/algorithms/action/tasks/train.py

```diff
@@ -157,14 +157,9 @@ def _train_model(self, dataset: DatasetEntity):
         self._model.train()
         # FIXME runner is built inside of train_model funciton, it is hard to change runner's type
         train_model(model=self._model, dataset=mm_train_dataset, cfg=training_config, validate=True)
-        checkpoint_file_path = glob(os.path.join(self._recipe_cfg.work_dir, "best*pth"))
+        checkpoint_file_path = glob(os.path.join(self._recipe_cfg.work_dir, "best*pth"))[0]
```

Comment:
Changes in otx.algorithms.action.tasks.train.py is to resolve mypy issue, same code with https://github.com/openvinotoolkit/training_extensions/pull/1956.